lib/rules/no-new-symbol.js

```diff
@@ -11,6 +11,8 @@
 
 module.exports = {
     meta: {
+        type: "suggestion",
```

Comment:
This seems like it should be a `problem`; the patterns reported by this rule will generally throw an error at runtime.

Reply:
Agreed